Launcher/MainWindow.xaml.cs

```diff
@@ -725,89 +725,57 @@ private void custom_run_Click(object sender, RoutedEventArgs e)
-                    }
-                    if (model_compile_type.HasFlag(model_compile.obj))
-                    {
-                        process.Arguments = "model-object " + path + "\\ " + obj;
-                        process.Arguments += " pause_after_run";
-                        RunProcess(process);
-                    }
-                }).Start();
-            }
-            if (model_compile_render_type.SelectedIndex == 1)
-            {
-                string path = compile_model_path.Text;
-                object_type obj = (object_type)model_compile_obj_type.SelectedIndex;
-                new Thread(delegate ()
+                    process.Arguments = "model-physics \"" + path + "\"";
+                    process.Arguments += " pause_after_run";
+                    RunProcess(process, true);
+                }
+                if (model_compile_type.HasFlag(model_compile.collision))
                 {
-                    var process = new ProcessStartInfo();
-                    process.WorkingDirectory = H2Ek_install_path;
-                    process.FileName = GetToolExeName(tool_type.tool);
-                    if (model_compile_type.HasFlag(model_compile.physics))
-                    {
-                        process.Arguments = "model-physics \"" + path + "\"";
-                        process.Arguments += " pause_after_run";
-                        RunProcess(process, true);
-                    }
-                    if (model_compile_type.HasFlag(model_compile.collision))
+                    process.Arguments = "model-collision \"" + path + "\"";
+                    process.Arguments += " pause_after_run";
+                    RunProcess(process, true);
+                }
+                if (model_compile_type.HasFlag(model_compile.render))
+                {
+                    if (render_type == 0)
                     {
-                        process.Arguments = "model-collision \"" + path + "\"";
+                        process.FileName = GetToolExeName(tool_type.tool);
+                        process.Arguments = "model-render \"" + path + "\"";
                         process.Arguments += " pause_after_run";
                         RunProcess(process, true);
                     }
-                    if (model_compile_type.HasFlag(model_compile.render))
+                    else
                     {
                         process.FileName = GetToolExeName(tool_type.daeconverter);
-                        process.Arguments = "-compile " + "data\\" + path;
-                        process.Arguments += " pause_after_run";
-                        RunProcess(process, true);
-                    }
-                    if (model_compile_type.HasFlag(model_compile.animations))
-                    {
-                        process.Arguments = "append-animations \"" + path + "\"";
+                        process.Arguments = "-compile " + "data\\" + path.Replace(H2Ek_install_path + "data\\", "");
```

Comment:
So DAE needs that modified path?

Reply:
Yes, doesn't seem to work otherwise from what I've tried.